src/core/thread/mle_router.cpp

```diff
@@ -2444,6 +2451,13 @@ otError MleRouter::HandleChildUpdateResponse(const Message &         aMessage,
 
     LogMleMessage("Receive Child Update Response from child", aMessageInfo.GetPeerAddr(), child->GetRloc16());
 
+    // Version
+    if (Tlv::GetTlv(aMessage, Tlv::kVersion, sizeof(version), version) == OT_ERROR_NONE)
```

Comment:
Thread 1.2 does not currently specify inclusion of Version TLV in Child Update Response.